src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -231,6 +231,8 @@ private static int getSchemaDelay()
 
     private final SamplingManager samplingManager = new SamplingManager();
 
+    private final boolean skipNotificationListeners = CassandraRelevantProperties.STORAGE_SERVICE_SKIP_NOTIFICATION_LISTENERS.getBoolean();
```

Comment:
@adelapena the logic is that I set that property _before_ StorageService is instantiated. So this will be just mere reader of that value. It is not meant to be mutable.